api/routes/api_teams.py

```diff
@@ -57,8 +58,8 @@ def get_players_endpoint():
     try:
         data = request.get_json()
         dto = TeamsStatisticsOrPlayersDTO(**data)
-        response = basketball_players(dto)
-        return get_good_response(response)
+        team_players = basketball_players(dto)
```

Comment:
Self_reminder: do we need to process different teams separately

Reply:
here must be another funk, but Vlad hasn't finished it yet